sopel/plugins/rules.py

```diff
@@ -814,6 +816,12 @@ def execute(self, bot, trigger):
         """
 
 
+class RateLimitCheckInfo(NamedTuple):
+    """Information from checking if a rate limit has been enforced"""
```

Comment:
```suggestion
    """Information from checking if a rate limit has been enforced."""
```

Nitpick early, nitpick often. 🙃